### DISCUSSION TITLE
有办法在别的网站调用memos的内容吗？

### DISCUSSION THREAD

**Discussion by beyondtop** [usememos/memos#110] [Q&A]
boss,有办法在别的网站调用memos的内容吗？应该如做？

**Comment by boojack** [comment]
> 调用 memos 的内容

指的是什么内容？通过 API 的形式调用吗？

**Reply by beyondtop** [reply]
是的，就是memos发布一条，就同步展示。

**Reply by rebron1900** [reply]
你可以通过api调用，js渲染。

**Reply by beyondtop** [reply]
添麻烦了。没有弄成功。

**Comment by lmm214** [comment]
https://immmmm.com/hi-memos/

https://www.life97.top/Dynamics.html
